server/neptune/sync/exec.go

```diff
@@ -0,0 +1,68 @@
+package sync
```

Comment:
Why in sync? this is meant to be concurrency related code.

How about just having an internal version of exec? or something like cmd? then the function can be called like `cmd.RunNewProcessGroup`

Reply:
moved to internal exec pkg